domains/transfer/Transfer.API/API/Models/CvrNumber.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace API.Models;
+
+public class CvrNumber : ValueObject
```

Comment:
You do not have to introduce ValueObject for this.

Reply:
corrected